src/model/Model.cpp

```diff
@@ -1,12 +1,11 @@
 #include "Model.h"
 #include <iostream>
 
-Model::Model(string name) {
-    tableName = name;
-}
+Model::Model(string name) :
+    tableName(name) {}
 
 Model::~Model() {
-    for (vector<Field*>::const_iterator it = fields.begin(); it != fields.end(); it++) {
+    for (vector<Field*>::const_iterator it = fields.begin(); it != fields.end(); ++it) {
```

Comment:
Yes, I copy/pasted the for-loops above from my test script. Only the value of `a` changed between the two.